core/src/main/java/hudson/slaves/DumbSlave.java

```diff
@@ -65,8 +65,8 @@ public DumbSlave(@Nonnull String name, String remoteFS, ComputerLauncher launche
         super(name, remoteFS, launcher);
     }
 
-    @Extension @Symbol({"dumb",
-            "slave", "agent" /*because this is in effect the canonical slave type*/})
+    @Extension @Symbol({"permanent" /*because this is in effect the canonical slave type*/, 
```

Comment:
"because this is in effect the canonical AGENT type"

Reply:
in this comment I understand "slave" as "hudson.model.Slave", not at the web UI concept